Makefile

```diff
@@ -319,9 +319,27 @@ teleport-hot-reload:
 		--exclude-dir="node_modules" \
 		--exclude-dir="target" \
 		--exclude-dir="web/packages/*/node_modules" \
+		--color \
+		--log-prefix=false \
 		--build="make $(BUILDDIR)/teleport" \
 		--command="$(BUILDDIR)/teleport $(TELEPORT_ARGS)"
 
+.PHONY: teleport-hot-reload-ent
+teleport-hot-reload-ent:
```

Comment:
Consider adding a `teleport-hot-reload` target to the e makefile instead. Then you can just run `make teleport-hot-reload` or `make -C e teleport-hot-reload` depending on what you want to run.

This will put less enterprise-specific knowledge into the OSS makefile.

Reply:
It felt a bit weird to create a target that jumps into a parent directory to watch it and then back again to build, but I guess these repos are tightly coupled anyway. I'll close this one and reopen it on the enterprise side.